paddle/gserver/layers/FeatureMapExpandLayer.cpp

```diff
@@ -62,6 +63,7 @@ bool FeatureMapExpandLayer::init(const LayerMap& layerMap,
 
   CHECK_EQ(inputLayers_.size(), 1UL);
   numFilters_ = config_.num_filters();
+  asRowVector_ = config_.user_arg() != "as_col_vec";
```

Comment:
`user_arg` had been defined in [ModelConfig.proto](https://github.com/PaddlePaddle/Paddle/blob/develop/proto/ModelConfig.proto#L441).